sql/select.go

```diff
@@ -468,13 +256,13 @@ func (v *indexInfo) init(s *scanNode) {
 
 // analyzeRanges examines the range map to determine the cost of using the
 // index.
-func (v *indexInfo) analyzeRanges(m qvalueRangeMap) {
+func (v *indexInfo) analyzeRanges(exprs []parser.Exprs) {
 	if !v.covering {
 		return
 	}
 
-	v.makeStartInfo(m)
-	v.makeEndInfo(m)
+	v.makeStartInfo(exprs)
```

Comment:
both of these methods do nothing if `len(exprs) != 1`; can `analyzeRanges` just take a single `expr`?